analysis_toolkit/compare.py

```diff
@@ -6,17 +6,17 @@
 
 def main(argv):
     try:
-        opts, args = getopt.getopt(argv, 'l')
+        opts, args = getopt.getopt(argv, "l")
```

Comment:
Oliko joku syy että ' -> " haluttaisiin? Tää on toki tottumiskysymys jne. mut tainnut muuten koodikanta olla kirjoitettuna yksöishipsuilla jota ainakin itse preferoin. Toisaalta aivan sama kun sitten formatter formatoi :shrug:

Reply:
> Suominäppiksellä vaan ' niin paljon helpompi ja kivempi kirjoittaa kuin shiftin vaativa "

Voi olla et täs projektissa on devaajia, joilla `"` ei vaadi shiftiä :thinking: Oletukset on vaarallisia. Tai ehkä ei niin vaarallisia tässä kohtaa kuitenkaan :zany_face: Tahdoin alunperin pitää `'` merkit koodikannassa, mut sit ku tutkin `black`in dokumentaatiota, ja näin että sen voi laittaa olemaan koskematta noihin merkkeihin vain komentorivioptiolla. Mut nyt kun muistutit `inv[oke]`n olemassaolosta, sehän ratkasee myös tämän ongelman :relaxed:

Ja vaikka nä muutokset saa piilotettua `git blame`sta, niin mun mielest silti ehkä vähän pare jos on pienempi delta. Suurin osa `black`in tekemistä formatointimuutoksist on just näit `'`-merkkejä.